Flaky test TestRaptorIntegrationSmokeTestBucketed.testCreateBucketedTable and TestRaptorMetadata.testCreateBucketedTable

```
[ERROR] Tests run: 1150, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1,405.751 s <<< FAILURE! - in TestSuite
[ERROR] testCreateBucketedTable(io.prestosql.plugin.raptor.legacy.TestRaptorIntegrationSmokeTestBucketed)  Time elapsed: 5.264 s  <<< FAILURE!
java.lang.AssertionError: 
For query: 
 SELECT count(DISTINCT "$shard_uuid") FROM orders_bucketed
not equal
Actual rows (up to 100 of 1 extra rows shown, 1 rows in total):
    [87]
Expected rows (up to 100 of 1 missing rows shown, 1 rows in total):
    [100]

	at org.testng.Assert.fail(Assert.java:94)
	at io.prestosql.tests.QueryAssertions.assertEqualsIgnoreOrder(QueryAssertions.java:237)
	at io.prestosql.tests.QueryAssertions.assertQuery(QueryAssertions.java:203)
	at io.prestosql.tests.QueryAssertions.assertQuery(QueryAssertions.java:106)
	at io.prestosql.tests.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:137)
	at io.prestosql.plugin.raptor.legacy.TestRaptorIntegrationSmokeTest.testCreateBucketedTable(TestRaptorIntegrationSmokeTest.java:451)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/BaseRaptorConnectorTest.java
+++ b/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/BaseRaptorConnectorTest.java
@@ -23,7 +23,6 @@ import io.trino.testing.MaterializedResult;
 import io.trino.testing.MaterializedRow;
 import io.trino.testing.TestingConnectorBehavior;
 import io.trino.testing.sql.TestTable;
-import io.trino.testng.services.Flaky;
 import org.intellij.lang.annotations.Language;
 import org.junit.jupiter.api.Test;
 
@@ -529,9 +528,6 @@ public abstract class BaseRaptorConnectorTest
     }
 
     @Test
-    @Flaky(
-            issue = "https://github.com/trinodb/trino/issues/1977",
-            match = "(?s)AssertionError.*query.*SELECT count\\(DISTINCT \"\\$shard_uuid\"\\) FROM orders_bucketed.*Actual rows.*\\[\\d\\d\\].*Expected rows.*\\[100\\]")
     public void testCreateBucketedTable()
     {
         assertUpdate("DROP TABLE IF EXISTS orders_bucketed");

--- a/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorBucketedConnectorTest.java
+++ b/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorBucketedConnectorTest.java
@@ -27,7 +27,7 @@ public class TestRaptorBucketedConnectorTest
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return createRaptorQueryRunner(ImmutableMap.of(), REQUIRED_TPCH_TABLES, true, ImmutableMap.of());
+        return createRaptorQueryRunner(ImmutableMap.of(), REQUIRED_TPCH_TABLES, true, ImmutableMap.of("storage.compaction-enabled", "false"));
     }
 
     @Test

--- a/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorConnectorTest.java
+++ b/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorConnectorTest.java
@@ -25,6 +25,6 @@ public class TestRaptorConnectorTest
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return createRaptorQueryRunner(ImmutableMap.of(), REQUIRED_TPCH_TABLES, false, ImmutableMap.of());
+        return createRaptorQueryRunner(ImmutableMap.of(), REQUIRED_TPCH_TABLES, false, ImmutableMap.of("storage.compaction-enabled", "false"));
     }
 }

--- a/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorMySqlConnectorTest.java
+++ b/plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/TestRaptorMySqlConnectorTest.java
@@ -72,6 +72,7 @@ public class TestRaptorMySqlConnectorTest
         Map<String, String> raptorProperties = ImmutableMap.<String, String>builder()
                 .put("metadata.db.type", "mysql")
                 .put("metadata.db.url", mysqlUrl)
+                .put("storage.compaction-enabled", "false")
                 .put("storage.data-directory", new File(baseDir, "data").getAbsolutePath())
                 .put("storage.max-shard-rows", "2000")
                 .put("backup.provider", "file")

